### ISSUE TITLE
Visual bug when editing a list

### ISSUE THREAD

**Issue by AvallonAB** [maruohon/tweakeroo#437]
When opening a window for list editing in tweakeroo a visual glitch appears. To be specific texts from other button that should be hidden under it render on top instead. I use `tweakeroo-fabric-1.20.1-0.17.0`
![2023-06-21_19 31 27](https://github.com/maruohon/tweakeroo/assets/52382664/ea6cc4ff-3b32-4f79-813e-6c21d020afed)

**Comment by ItzSkyReed** [comment]
Ye.. This is annoying

**Comment by maruohon** [comment]
Yeah it broke again in the 1.20 update... I just hack fixed these long standing gui rendering bugs in 1.19.4 and now this is back again >_>

But the plan now is not to touch this crap again:
![image](https://github.com/maruohon/tweakeroo/assets/2331050/a6556981-5dd4-4484-9c2c-6208c317522c)

![image](https://github.com/maruohon/tweakeroo/assets/2331050/e4752e1d-0d69-411b-92cf-2ccc2f946170)
